azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/OfferImpl.java

```diff
@@ -7,7 +7,7 @@
 import com.microsoft.azure.management.resources.fluentcore.arm.Region;
 
 /**
- * The implementation for {@link VirtualMachineImage.Offer}.
+ * The implementation for {@link Offer}.
```

Comment:
I wonder if it is possible to add this enforcement to checkstyle: to look for javadocs that reference non existent types...